### PR TITLE
Adding Cadence Support to Terraform Provider

### PR DESCRIPTION
**Pull request by RussDavidson-InstaClustr**

### CODE REVIEW

**Review by arjunrajinstaclustr — DISMISSED** [review]

**Review by tharinduamila-insta — DISMISSED** [review]

File: instaclustr/structs.go [hunk 1]
```diff
@@ -89,6 +96,7 @@ type CreateRequest struct {
 	ClusterNetwork        string                    `json:"clusterNetwork,omitempty"`
 	PrivateNetworkCluster string                    `json:"privateNetworkCluster,omitempty"`
 	PCICompliantCluster   string                    `json:"pciCompliantCluster,omitempty"`
+	NeedsLoadBalancer     string                    `json:"needsLoadBalancer,omitempty"`
```
[the remark this answers is not in the document]
Comment (RussDavidson-InstaClustr) [reply instaclustr/structs.go:99]: I'd followed the same approach that was used for the other variables there that are booleans, but declared as strings there too, such as PrivateNetworkCluster and PCICompliantCluster.

[the remark this answers is not in the document]
Comment (arjunrajinstaclustr) [reply instaclustr/structs.go:99]: Don't know if we did that intentionally or on purpose to avoid some bug.

**Review by arjunrajinstaclustr — DISMISSED** [review]

**Review by tharinduamila-insta — APPROVED** [review]

File: instaclustr/structs.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (RussDavidson-InstaClustr) [reply instaclustr/structs.go]: This property has now been removed, as of yesterday it is no longer required to be sent to the API ad the default behaviour has been undated to be true in the API for Cadence clusters.

**Comment by RussDavidson-InstaClustr** [comment]
Redoing

**Pull request closed without merging**